### ISSUE TITLE
After the bootrapping of the normal installation "No OVA" Steps 1 does not start

### ISSUE THREAD

**Issue by Beowulf67** [EMCECS/ECS-CommunityEdition#386]
### Expected Behavior
Step1 should start and run

### Actual Behavior
Step1 does not start
Got errory -Unable to load any data from source yaml file
### Steps to Reproduce Behavior
restart step1

### Output

[admin@ifclecs02s02n01 ~]$ cd ECS-CommunityEdition/
[admin@ifclecs02s02n01 ECS-CommunityEdition]$ step1
[sudo] password for admin:
ecs-install> Applying deploy.yml
Unable to load any data from source yaml file
Unable to load any data from source yaml file
>

**Comment by Beowulf67** [comment]
Hm its was easy to fix
I just copied a new "not configured" deploy.yml
cp docs/design/reference.deploy.yml deploy.yml
and edit this one, worked.
Just creating a new one with vi deploy.yml and copied my edited Version into it did not work.

**Comment by obalkan** [comment]
Thanks, your information was helpful for me.

**Comment by padthaitofuhot** [comment]
Yes, a deploy.yml file must be created.  It is helpful to use the template at `docs/design/reference.deploy.yml`.  Perhaps the documentation could be made more clear in this aspect.

**Comment by Beowulf67** [comment]
Hi All
 The Trap i fall in was to create a new  one directly with the vi editor. and then copied in the values/config i build with a wordpad before. And saved it before.  May thats the reason why it happend. I did it this way  around 10 times and always had the same Problem. After i followed the documentation as described, it worked.
May someone can test it when he setup a new one?  I will check if there are differences with the lf or crlf in both versions. Thats the only point I can currently imagine.
Thanks



Von meinem Samsung Galaxy Smartphone gesendet.


-------- Ursprüngliche Nachricht --------
Von: Travis Wichert <notifications@github.com>
Datum: 14.11.17 17:45 (GMT+01:00)
An: EMCECS/ECS-CommunityEdition <ECS-CommunityEdition@noreply.github.com>
Cc: Jörg Pieper <joerg.pieper@logicalis.de>, State change <state_change@noreply.github.com>
Betreff: Re: [EMCECS/ECS-CommunityEdition] After the bootrapping of the normal installation "No OVA" Steps 1 does not start (#386)


Yes, a deploy.yml file must be created. It is helpful to use the template at docs/design/reference.deploy.yml. Perhaps the documentation could be made more clear in this aspect.

**Comment by Beowulf67** [comment]
Sorry compared both files. 
There is no difference when i compare them line by line.
Also both have a LF at the end of each line,
Currenly I have no clue why the one is working the other is not.
May some can bring some light into this?

**Comment by padthaitofuhot** [comment]
I have no idea.  I haven't been able to duplicate this issue at all.  Can you zip up your two different deploy.yml files and email them to me at padthaitofuhot@gmail.com ?  Thanks!

**Comment by JellevdK** [comment]
If the yaml syntax in deploy.yml is broken (because for example  a wrong identation) then these errors are displayed. I think it should be very usefull that the `videploy` tool also does yaml syntax checking.

**Comment by padthaitofuhot** [comment]
YAML syntax checking is performed when deploy.yml is performed.  It's not failing syntax checking.
